src/micromamba/devcontainer-feature.json

```diff
@@ -9,12 +9,26 @@
             "type": "string",
             "proposals": [
                 "latest",
-                "1"
+                "1.0.0"
             ],
             "default": "latest",
-            "description": "Select version of micromamba."
+            "description": "Exact version of Micromamba to install (must be X.Y.Z)"
+        },
+        "reinstall": {
+            "type": "boolean",
+            "default": "false",
+            "description": "Reinstall in case Micromamba already exists"
+        },
+        "addCondaForge": {
+            "type": "boolean",
+            "default": false,
+            "description": "Add conda-forge channel to the config?"
```

Comment:
I question this setup as I want to use only conda-forge.
Is there any guidance from the mamba community?

Reply:
If using the Anaconda repository, we should notify users like `ghcr.io/devcontainers/features/conda`.
https://github.com/devcontainers/features/tree/main/src/conda

Reply:
I and everyone I know only uses conda-forge. If it were up to me, I would use conda-forge as default.

Mamba abstains from setting a default channel, I believe for purpose of neutrality.

I was not planning to offer Anaconda. Do you think we should?

Reply:
Sorry, I misunderstood that Anaconda Repository is registered by default.
It is definitely reasonable and desirable that no channels are registered by default. However, it may be worth mentioning the default state in `NOTES.md`.
Since it is the same option name of `ghcr.io/devcontainers/features/conda`'s, users may expect the channel to be registered from the beginning like `ghcr.io/devcontainers/features/conda` or `ghcr.io/rocker-org/devcontainer-features/miniforge`.

Reply:
Maybe we should replace the `addCondaForge` argument with `channels`, which would be a comma(?)-delimited list?

Reply:
> Maybe we should replace the `addCondaForge` argument with `channels`, which would be a comma(?)-delimited list?

Good idea!
A comma-separated string is best for now. (See https://github.com/devcontainers/spec/issues/57#issuecomment-1309015728)

Perhaps set `conda-forge` to default value?
If you do not set a default, you should include an example in `NOTES.md`, like this.
https://github.com/rocker-org/devcontainer-features/blob/main/src/apt-packages/README.md

Reply:
Done.